lib/common/value-utils.js

```diff
@@ -0,0 +1,51 @@
+'use strict'
+
+const { nativeImage, NativeImage } = process.atomBinding('native_image')
+
+const objectMap = function (value, mapper) {
+  const object = {}
+  for (const key of Object.keys(value)) {
+    object[key] = mapper(value[key])
+  }
+  return object
+}
```

Comment:
The in-loop object lookups can be avoided with `Object.entries()`

```js
const objectMap = function (source, mapper) {
  const target = {}
  for (const [key, val] of Object.entries(source)) {
    target[key] = mapper(val)
  }
  return target
}
```

Reply:
or even
```
const objectMap = function (object, mapper) {
  return Object.fromEntries(Object.entries(object).map(([ key, value ]) => [ key, mapper(value) ]))
}
```